### ISSUE TITLE
Where are the Kotlin communities and enthusiasts?

### ISSUE THREAD

**Issue by kytrinyx** [exercism/kotlin#132]
As we move towards the launch of the new version of Exercism we are going to be ramping up on actively recruiting people to help provide feedback.

Our goal is to get to 100%: everyone who submits a solution and wants feedback should get feedback. Good feedback. You can read more about this aspect of the new site here: http://mentoring.exercism.io/

To do this, we're going to need a lot more information about where we can find language enthusiasts.

* Is Kotlin supported by one or more large organizations?
* Does Kotlin have an official community manager?
* Do you know of specific communities (online or offline) that are enthusiastic about Kotlin? (Chat communities, forums, meetups, student clubs, etc)
* Are there popular conferences for Kotlin? (If so, what are some examples?)
* Are there any organizations who are targeted specifically at getting certain subgroups or demographics interested in Kotlin? (e.g. kids, teenagers, career changers, people belonging to various groups that are typically underrepresented in tech?)
* Are there specific groups or programs dedicated to mentoring people in Kotlin?
* Are there popular newsletters for Kotlin?
* Is Kotlin taught at programming bootcamps? (If so, what are some examples?)
* Is Kotlin taught at universities? (If so, what are some examples?)

In other words: where do people care a lot and/or know a lot about Kotlin?

This is part of the project being tracked in https://github.com/exercism/meta/issues/103

**Comment by roger-fang** [comment]
Kotlin was created and is maintained by Jetbrains, the company that developed the IntelliJ IDE and Android Studio, and is licensed under the Apache 2 license. It’s been under development since 2010 and had its first public release in 2012. It runs on the JVM, can be compiled for Android or native binary, and can be transpiled to JavaScript.

In 2017, at Google I/O developers conference, Google announced official support for Kotlin for Android development. Many companies are switching from Java to Kotlin for new Android development.
It's used by Pinterest, Gradle, Evernote, Uber, Corda, Coursera, Pivotal (Spring Framework), and Atlassian.

Kotlin is taught at Metro State University in Denver for their Android Programming course.

Kotlin online forum: [https://discuss.kotlinlang.org/](url)
List of links to Kotlin user groups: [https://kotlinlang.org/community/user-groups.html](url)

[https://kotlinlang.org/](url)

**Comment by kytrinyx** [comment]
This is super useful, thanks!
